### ISSUE TITLE
Still aria-foo mismatches

### ISSUE THREAD

**Issue by StommePoes** [Pearson-Higher-Ed/text-modal#2]
In src/js/component-owner.js

We still have 

```
<div id="modalContent" className="modalContent" aria-labelledby="modalContent">
```

This code is saying that this div is labelling itself with itself. It shouldn't be doing that, at least not out in public.

instead, the outer <Modal> element (whoever has the role="dialog" on them) should have aria-labelledby pointing to the heading's ID (in this case `modalHeaderText`).

Additionally the footer is still also referring to itself:

```
<div id="modalFooter" className="modalFooter" aria-labelledby="modalFooter">
```

Footer doesn't need any labels, and if it only has an ID for aria, then the id can go away as well (same goes for the body section's id).

**Comment by JasonLantz** [comment]
just pushed a change that adds this line on open:

document.querySelectorAll('[role="dialog"]')[0].setAttribute('aria-labeledby', 'modalHeaderText');
